capa/features/extractors/ida/helpers.py

```diff
@@ -26,15 +27,19 @@ def find_byte_sequence(start: int, end: int, seq: bytes) -> Iterator[int]:
         end: max virtual address
         seq: bytes to search e.g. b"\x01\x03"
     """
+    patterns = ida_bytes.compiled_binpat_vec_t()
+    encoding = ida_nalt.get_default_encoding_idx(ida_nalt.BPU_1B)
```

Comment:
Please add a comment as to why `ida_nalt.BPU_1B` is used here.

Reply:
When preparing for a `bin_search`, we could use a one byte-per-character encoding (i.e. ida_nalt.BPU_1B), or opt for UTF-16 instead (ida_nalt.BPU_2B). UTF-32 is also available via `ida_nalt.BPU_4B`

Do you suggest I should change the encoding to UTF-16 instead of UTF-8?

Reply:
Got it- please add a comment in the code explaining this, e.g. `# use one byte-per-character encoding`, making it obvious what `ida_nalt.BPU_1B` means.